src/libcommonserver/io/iohelper_mac.mm

```diff
@@ -153,13 +154,13 @@ IoError nsError2ioError(NSError *nsError) noexcept {
     data = std::string(reinterpret_cast<char const *>(buffer), size);
     delete[] buffer;
 
+    // Read target
     Boolean isStale;
     CFURLRef targetUrl =
         CFURLCreateByResolvingBookmarkData(nil, bookmarkRef, kCFBookmarkResolutionWithoutUIMask, nil, nil, &isStale, &error);
     CFRelease(bookmarkRef);
     if (targetUrl == nil) {
         if (error) {
-            ioError = nsError2ioError((NSError *)error);
```

Comment:
So we ignore this error now?

Reply:
Yes. When the target path of the alias cannot be read, an empty value is sent to the back, it's not an issue as it is not used later to create the alias (see DownloadJob::createLink).
UploadJob::setQueryParameters has been modified to support sending an empty target path.